src/cli.ts

```diff
@@ -60,7 +60,13 @@ yargs()
 			description: 'Watch mode',
 			group: INSTANT_MOCHA_OPTIONS_GROUP,
 		},
+		mode: {
+			description: 'Mode passed to webpack development|production',
+			group: INSTANT_MOCHA_OPTIONS_GROUP,
+			type: 'string',
+		},
```

Comment:
Not sure how I feel about mixing mocha and webpack CLI arguments. I think I prefer to keep it just mocha CLI args and bare-minimum instant-mocha args.

Reply:
I think that mode flag for webpack is a pretty big flag as my config will add bundle minimizers and the flag itself can change how webpack behaves for a production vs development environment build. I think its important to test your production code  when you are running the tests against the new build, but when developing new tests I would rather have the development mode enabled to have faster build times.

Reply:
I went ahead and split up the commits to separate the flag and the webpack config function changes.

Reply:
Gotcha. I was thinking most use-cases would have a separate Webpack config for tests with develop mode hard-coded, but perhaps that's not the case.

Let's add it in.

Reply:
yea I went the route of having my config being just a function that changes the config based on the flag mainly. Have a couple checks for if test environment variable is set, but that is just on process.env